superset-frontend/src/dashboard/containers/Chart.jsx

```diff
@@ -83,6 +83,8 @@ function mapStateToProps(
     formData,
     editMode: dashboardState.editMode,
     isExpanded: !!dashboardState.expandedSlices[id],
+    showFilterBadge: dashboardInfo.metadata?.showFilterBadge ?? true,
+    showCrossFilterBadge: dashboardInfo.metadata?.showCrossFilterBadge ?? true,
```

Comment:
I think it's better to do `hydrate` file default value

Reply:
Yes, let's pick this up in the hydration phase.